CHANGELOG.md

```diff
@@ -18,6 +18,8 @@ the detailed section referring to by linking pull requests or issues.
 
 #### Minor
 
+- Implemented markdown for asset descriptions
```

Comment:
Could we maybe fade into white instead?

![grafik](https://github.com/sovity/edc-ui/assets/30114507/24defe22-ce93-4785-843d-e2e5afb437f4)

Reply:
Yes it looks better imo